apps/etterlatte-behandling/src/main/kotlin/behandling/BrukerService.kt

```diff
@@ -48,14 +42,20 @@ class BrukerServiceImpl(
             }
 
             else -> {
-                val geografiskTilknytning = tilknytning.geografiskTilknytning() ?: throw GeografiskTilknytningMangler()
-                if (tilknytning.harBareLandTilknytning()) {
-                    if (tilknytning.land!! == "NO") {
-                        Navkontor(navn = "UKjent kontor, men har Norge som landkode", enhetNr = "ukjent enhetsnummer")
+                val geografiskTilknytning = tilknytning.geografiskTilknytning()
+                when {
+                    tilknytning.harBareLandTilknytning() -> {
+                        if (tilknytning.land!! == "NO") {
+                            Navkontor(navn = "UKjent kontor, men har Norge som landkode", enhetNr = "ukjent enhetsnummer")
+                        }
```

Comment:
```suggestion
                        if (tilknytning.land == "NO") {
                            Navkontor(navn = "Ukjent kontor, men har Norge som landkode", enhetNr = "ukjent enhetsnummer")
                        }
```

Kan vi potensielt få noen konsekvenser av å sette `enhetNr` til noe annet enn 4 siffer? 🤔

Reply:
Nope, vi bruker bare navnet i frontend uansett.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3524/commits/d0c75783308c9792623ae6b6746b2f1ca791c402